<launch>

    <arg name="launch_prefix"       default=""/>
    <arg name="imu_topic"           />
    <arg name="pose_topic"          />
    <arg name="transform_input"     />
    <arg name="param_file"          default="$(find msf_updates)/pose_sensor_fix.yaml"/>

    <node  launch-prefix="$(arg launch_prefix)" name="msf_pose_sensor" pkg="msf_updates" type="pose_sensor" clear_params="true" output="screen">
          <remap from="msf_core/imu_state_input" to="$(arg imu_topic)"/>
          <!-- <remap from="msf_updates/pose_with_covariance_input" to="$(arg pose_topic)" /> -->
          <remap from="msf_updates/transform_input" to="$(arg transform_input)" />
          <rosparam file="$(arg param_file)"/>
    </node>
</launch>
